tcms/testplans/static/testplans/js/mutable.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready((function () {
+    populateProductVersion();
+
+    $('#id_product').change(populateProductVersion);
+
+    $('#add_id_product').click(function () {
+        return showRelatedObjectPopup(this);
+    });
+
+    $('#add_id_version').click(function () {
+        return showRelatedObjectPopup(this);
+    });
+}));
+
+function populateProductVersion() {
+    let href = $('#add_id_version')[0].href;
+    $('#add_id_version')[0].href = href.slice(0, href.indexOf('&product'));
+
+    $('#id_version').find('option').remove();
+    $('#add_id_version')[0].href += `&product=${$('#id_product').val()}`;
```

Comment:
swap these 2 lines so that the operations with the href attribute are grouped together.